lib/services/bootstrap/internalBootstrap.js

```diff
@@ -27,13 +27,14 @@ const
   errorsManager = require('../../config/error-codes/throw');
 
 class InternalBootstrap extends SafeBootstrap {
-  constructor (kuzzle) {
+  constructor (kuzzle, storageEngine) {
```

Comment:
Please do not review these changes, this class will be adapted to use the new InternalStorage in another PR